src/test/java/org/apache/drillui/test/framework/pages/BasePage.java

```diff
@@ -119,6 +117,31 @@ protected boolean isElementStable(WebElement element) {
     return position.equals(element.getLocation());
   }
 
+  protected void forceClick(WebElement element) {
```

Comment:
Please remove the method as it is a duplicate.

Reply:
Done